charts/datahub/README.md

```diff
@@ -87,6 +87,7 @@ helm install datahub datahub/datahub --values <<path-to-values-file>>
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| acryl-datahub-actions.ingestionSecretFiles.name | string | `""` | Name of the k8s secret that holds any secret files (e.g., SSL certificates and private keys) that are used in your ingestion recipes |
```

Comment:
Please add the path under which the secret values will be stored: `/etc/datahub/ingestion-secret-files`